common/models/user.js

```diff
@@ -312,6 +312,7 @@ module.exports = function(User) {
       ctx.where[pkName] = { inq: ids };
 
       AccessToken.destroyAll({ userId: { inq: ids }}, next);
+      console.log('tokens deleted for users %j', ids);
```

Comment:
I always remove as I did within the previous PR. I just need it for testing. 
It is not even safe in production mode, so I won't keep it at all.